src/Symfony/Bundle/WebProfilerBundle/Resources/views/Icon/form.svg.twig

```diff
@@ -0,0 +1,6 @@
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" x="0px" y="0px" height="24" viewBox="0 0 24 24" enable-background="new 0 0 24 24" xml:space="preserve">
```

Comment:
these files are pure SVG files, with no twig markup in them. I suggest removing the ``.twig`` extension on them. This way, tools will consider them as SVG files (github will probably show a preview for instance).
Twig is able to include any file anyway (the requirement for a ``.twig`` extension comes from the Templating component to support multiple engines, not from Twig, and we don't use the Templating component for these templates)